src/tiled/mainwindow.cpp

```diff
@@ -2501,6 +2493,12 @@ void MainWindow::currentEditorChanged(Editor *editor)
         statusBar()->removeWidget(widget);
     mEditorStatusBarWidgets.clear();
 
+    // Somehow, in Qt 6 the above hiding of widgets can hide the news button
+#if QT_VERSION > QT_VERSION_CHECK(6, 0, 0)
```

Comment:
Not that it matters in practice since nobody is going to build Tiled against Qt 6.0.0, and I don't actually know since when this bug exists, but I meant to check `>=` here.